components/tyk-dashboard/values.yaml

```diff
@@ -193,6 +193,15 @@ dashboard:
   # The hostname to bind the Dashboard to.
   hostName: tyk-dashboard.local
 
+  tls:
+    # When true, it will install the certificate present in the templates folder, set to false when using
+    # a custom TLS certificate to avoid overwriting yours
+    useDefaultTykCertificate: true
```

Comment:
We no longer need this field since `/certs` is deleted. Also, other occurrences of this field should be deleted. Can you please check it?

Reply:
I only deleted the `/certs` folder because I am using the same certs across all the components. The gateway creates those secrets and I use across. Maybe there is a better way to do it?

Reply:
So what I decided to do is to move `useDefaultTykCertificate` under `global.tls` because this is shared across all components when using self-signed certs